pkg/storaged/iscsi-panel.jsx

```diff
@@ -226,7 +226,11 @@ export class IscsiPanel extends React.Component {
                 );
 
             return {
-                client, actions, kind: "array", name: session.data.target_name || "", key: path,
+                client,
+                actions,
+                kind: "array",
+                name: session.data.target_name || "",
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-17890-20221108-090107-5c6c50b0-fedora-36-devel//Coverage/lcov/github-pr.diff.gcov.html)